modules/sdk/gradle-plugins-defaults/src/main/java/com/liferay/gradle/plugins/defaults/LiferayAppDefaultsPlugin.java

```diff
@@ -192,6 +194,8 @@ private void _configureAppJSDoc(Project project, Project privateProject) {
 			return;
 		}
 
+		GradleUtil.applyPlugin(project, NodeDefaultsPlugin.class);
```

Comment:
Please move this and all other `GradleUtil.applyPlugin` invocations into the `_applyPlugins` method.